src/components/sdk/DeviceSelection/CameraSelection/VideoFilterSelection.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { ChangeEvent, useEffect, useState } from 'react';
+
+import { FormField } from '../../../ui/FormField';
+import { Select } from '../../../ui/Select';
+import { useBackgroundBlur } from '../../../../providers/BackgroundBlurProvider';
+import { useMeetingManager } from '../../../../providers/MeetingProvider';
+import { isVideoTransformDevice } from 'amazon-chime-sdk-js';
+
+interface Props {
+  /** Label shown for video filter selection, by default it is "Video Filter" */
+  label?: string;
+  /** Label shown in the dropdown when no video filter has been selected yet, by default it is "Select video filter" */
+  labelForUnselected?: string;
+}
+
+export const VideoFilterSelection: React.FC<Props> = ({
+  label = 'Video Filter',
+  labelForUnselected = 'Select video filter',
+}) => {
+  const {isBackgroundBlurSupported, createBackgroundBlurDevice} = useBackgroundBlur();
+  const [filterName, setFilterName] = useState('unselected');
+  const meetingManager = useMeetingManager();
+  const filterOptions = [
+    {
+      label: labelForUnselected,
+      value: 'unselected',
+    },
+    {
+      label: 'Background Blur',
+      value: 'Background Blur'
+    }
+  ];
+
+
+  async function selectFilter(e: ChangeEvent<HTMLSelectElement>) {
```

Comment:
Since there is one value, I don't think we should use a selection here. Please consult UX but it should be just a checkbox somewhere.

Reply:
okay - changed to a checkbox per Jaimes feedback